src/components/TouristInfoCards.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+function TouristInfoCards() {
+  return (
+    <div className="container">
+      <div className="card">
```

Comment:
The code responsible for creating a single card is repeated multiple times here. It's a sign that we should create a new `Card` component to reuse that code. My suggestion on how to tackle this would be to:
- create an array of objects that contain data for each city such as link, image source and summary
- map over that array and for each element create a Card component, passing data to that component with props